cmd/result-aggregator/main.go

```diff
@@ -5,6 +5,8 @@ import (
 	"flag"
 	"fmt"
 	"io/ioutil"
+	"k8s.io/test-infra/prow/config/secret"
+	"k8s.io/test-infra/prow/logrusutil"
```

Comment:
imports

Reply:
can we please not worry about this? it's mindless silly work that makes no impact on anything -- my editor doesn't even show me the import stanza